build.gradle

```diff
@@ -4,15 +4,14 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.1.3'
+    classpath 'com.android.tools.build:gradle:3.3.2'
```

Comment:
You'll have to update the CI script to install these dependencies.

Reply:
I will try to update the script!